Bump files with dotnet-file sync

kzu opened the pull request:
# devlooped/oss

- Update combine prs default message https://github.com/devlooped/oss/commit/74189b0
- Fix syntax for excluding one file https://github.com/devlooped/oss/commit/5d05e54
- Use BOT_ defaults consistently https://github.com/devlooped/oss/commit/98919f7
- Use simple bash variable expansion https://github.com/devlooped/oss/commit/acedd1d
- Ensure checkout before defaults https://github.com/devlooped/oss/commit/721ee85
- Rework to pass secrets explicitly https://github.com/devlooped/oss/commit/15e9486
- Update to public bot defaults action https://github.com/devlooped/oss/commit/b9671b9
- Add fallback GITHUB_TOKEN to bot defaults https://github.com/devlooped/oss/commit/5406d90
- Use bot account for sync commit author https://github.com/devlooped/oss/commit/1d3a2f4
- Ensure both author and committer match https://github.com/devlooped/oss/commit/d94ddb1
- Resolve includes after file sync https://github.com/devlooped/oss/commit/8f45cf2
- By default don't validate includes https://github.com/devlooped/oss/commit/aed791a
- Ignore errors creating the PR https://github.com/devlooped/oss/commit/b97b8f1
- Always pass in auth headers to GH API https://github.com/devlooped/oss/commit/a922d03
- Since dependabot doesn't consume API requests, do it more frequently https://github.com/devlooped/oss/commit/4f070a4
- Move format check for last https://github.com/devlooped/oss/commit/7db501b
- Make build matrix configurable per-repo https://github.com/devlooped/oss/commit/391da5e
- Rename matrix lookup job and steps https://github.com/devlooped/oss/commit/cf8e339

# devlooped/.github

- Simplify workflow, skip for bot accounts https://github.com/devlooped/.github/commit/e347e5c

# devlooped/catbag

- :bug: Fix xmldoc https://github.com/devlooped/catbag/commit/ebbc311
- Fix xmldoc https://github.com/devlooped/catbag/commit/53dd653

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/devlooped/html?pullRequest=11) <br/>All committers have signed the CLA.

kzu committed:
⬆️ Bump files with dotnet-file sync
# devlooped/oss

- Update combine prs default message https://github.com/devlooped/oss/commit/74189b0
- Fix syntax for excluding one file https://github.com/devlooped/oss/commit/5d05e54
- Use BOT_ defaults consistently https://github.com/devlooped/oss/commit/98919f7
- Use simple bash variable expansion https://github.com/devlooped/oss/commit/acedd1d
- Ensure checkout before defaults https://github.com/devlooped/oss/commit/721ee85
- Rework to pass secrets explicitly https://github.com/devlooped/oss/commit/15e9486
- Update to public bot defaults action https://github.com/devlooped/oss/commit/b9671b9
- Add fallback GITHUB_TOKEN to bot defaults https://github.com/devlooped/oss/commit/5406d90
- Use bot account for sync commit author https://github.com/devlooped/oss/commit/1d3a2f4
- Ensure both author and committer match https://github.com/devlooped/oss/commit/d94ddb1
- Resolve includes after file sync https://github.com/devlooped/oss/commit/8f45cf2
- By default don't validate includes https://github.com/devlooped/oss/commit/aed791a
- Ignore errors creating the PR https://github.com/devlooped/oss/commit/b97b8f1
- Always pass in auth headers to GH API https://github.com/devlooped/oss/commit/a922d03
- Since dependabot doesn't consume API requests, do it more frequently https://github.com/devlooped/oss/commit/4f070a4
- Move format check for last https://github.com/devlooped/oss/commit/7db501b
- Make build matrix configurable per-repo https://github.com/devlooped/oss/commit/391da5e
- Rename matrix lookup job and steps https://github.com/devlooped/oss/commit/cf8e339

# devlooped/.github

- Simplify workflow, skip for bot accounts https://github.com/devlooped/.github/commit/e347e5c

# devlooped/catbag

- :bug: Fix xmldoc https://github.com/devlooped/catbag/commit/ebbc311
- Fix xmldoc https://github.com/devlooped/catbag/commit/53dd653
